exercises/isogram/Cargo.toml

```diff
@@ -0,0 +1,3 @@
+[package]
+name = "isogram"
+version = "0.1.0"
```

Comment:
https://github.com/exercism/problem-specifications/blob/master/exercises/isogram/canonical-data.json#L6 is at 1.2.0. So, the version here should also be at 1.2.0 and use the cases listed in https://github.com/exercism/problem-specifications/blob/master/exercises/isogram/canonical-data.json , OR write in this PR a comment explaining why those cases are not appropriate for use in the Rust track.